docs/PLUGINS.md

```diff
@@ -174,30 +177,31 @@ default values as a Desciptor is a mutable object, i.e. data-binding won't reset
 ```
 
 ### Step 6
-If you don't have one already, define a `@Symbol` annotation on your descriptor. This is the name end-user will be able to use to access
-your Descriptor for configuration. To avoid collisions with other plugin, prefer using your plugin's artifactId as a symbolic name for your
+
+If you don't have one already, define a `@Symbol` annotation on your descriptor. This is the name an end user will be able to use to access
+your Descriptor for configuration. To avoid collisions with other plugins, prefer using your plugin's artifactId as a symbolic name for your
```

Comment:
```suggestion
your `Descriptor` for configuration. To avoid collisions with other plugins, prefer using your plugin's artifactId as a symbolic name for your
```